topics/go/algorithms/strings/reverse/reverse.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import "fmt"
+
+func main() {
+	// Run ReverseString
```

Comment:
I need complete sentences and proper grammar and puncuation on all comments. Plus I need a line space above each comment. Ex

// Run the reverse string function.

Now that being said, let's make sure the comments are really providing extra information that is not obvious in the code. This comment might not be adding value.

Reply:
Thanks. I have removed some comments and added others.